README.md

```diff
@@ -30,3 +30,72 @@ Once you've set up nixops first run `./generate_secrets.sh` then continue with t
 
 At the moment this relies on using the unstable nixpkgs channel.
 The "all" profile requires 15 GB of disk space and 2GB of memory.
+
+**For example, on Debian 9 Stretch:**
+
+Install Dependencies: 
+```
+sudo apt-get install curl git gnupg2 dirmngr
+```
+Install Latest Nix with GPG Verification:
+```
+curl -o install-nix-2.1.3 https://nixos.org/nix/install
+curl -o install-nix-2.1.3.sig https://nixos.org/nix/install.sig
+gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE
+gpg2 --verify ./install-nix-2.1.3.sig
+sh ./install-nix-2.1.3
+. /home/user/.nix-profile/etc/profile.d/nix.sh
+```
+Add virtualbox.list to /etc/apt/sources.list.d
+```
+deb http://download.virtualbox.org/virtualbox/debian stretch contrib
+```
+Add Oracle VirtualBox public key:
+```
+wget https://www.virtualbox.org/download/oracle_vbox_2016.asc
+sudo apt-key add oracle_vbox_2016.asc
+```
+Install virtualbox-5.2
+```
+sudo apt-get update
+sudo apt-get install virtualbox-5.2
+```
+Build Nixops from source
+```
+git clone https://github.com/NixOS/nixops
+cd ~/nixops
+nix-build release.nix -A build.x86_64-linux
+cd
+```
+This should output a last line like `/nix/store/wa6nk3aqxjb2mgl9pkwrnawqnh9z1b9d-nixops-1.6.1pre0_abcdef/`. This is the directory Nixops is installed in. Note it for later.
+
+Create Host Adapter in VirtualBox:
+```
+Open VirtualBox
+File -> Host Network Manager -> Create
+This should create a hostadapter named vboxnet0
+```
+Clone this project
+```
+cd
+git clone https://github.com/jonasnick/nix-bitcoin
+cd ~/nix-bitcoin
+```
+Generate Secrets
+```
+./generate_secrets.sh
+```
+Create Nixops
+```
+nixops create network.nix network-vbox.nix -d bitcoin-node
+```
+Replace `nixops` with the path to the nixops you built from source. For example: `/nix/store/wa6nk3aqxjb2mgl9pkwrnawqnh9z1b9d-nixops-1.6.1pre0_abcdef/bin/nixops`
+
+Deploy Nixops
+```
+nixops  deploy -d bitcoin-node
+```
+Replace `nixops` with the path to the nixops you built from source. For example: `/nix/store/wa6nk3aqxjb2mgl9pkwrnawqnh9z1b9d-nixops-1.6.1pre0_abcdef/bin/nixops`
```

Comment:
It could be mentioned that you can change your path, i.e. `export PATH=/nix/store/wa6nk3aqxjb2mgl9pkwrnawqnh9z1b9d-nixops-1.6.1pre0_abcdef/bin/:$PATH` so you can just type `nixops`.